src/UserSwitch/UserSwitch.tsx

```diff
@@ -95,7 +94,7 @@ const UserSwitch: FC<Props> = ({
   const [anchorEl, setAnchorEl] = useState<(EventTarget & Element) | null>(
     null,
   );
-  const memberName = member.get('name');
+  const memberName = member?.name;
```

Comment:
I think now you can desctructure? It might be a nice refactor to avoid all `member.xx`.